tests/bluetooth/tester/src/gap.c

```diff
@@ -0,0 +1,269 @@
+/* gap.c - Bluetooth GAP Tester */
```

Comment:
Are the GAP commands necessary to run the MMDL test suite?

Reply:
I left only what was necessary without moving the code around. This way there is no need to modify autopts code. In `gap_init()` we call `bt_enable()` and in `gap_read_ctrl_info()` we set an identity address and communicate it to PTS. This is how it was designed and while I think it would be nice to change it, we would have to do this in upstream (decouple gap and mesh).

Reply:
Alright, makes sense. Keep it :)